lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/ChunkedReadWriteHandler.java

```diff
@@ -170,8 +170,8 @@ private int getSize(final byte[] data, final int offset) {
       return 0;
     }
 
-    final ByteBuf intBuffer = Unpooled.wrappedBuffer(data, offset, INT_SIZE).order(Unpooled.LITTLE_ENDIAN);
-    final int ret = intBuffer.readInt();
+    final ByteBuf intBuffer = Unpooled.wrappedBuffer(data, offset, INT_SIZE);
```

Comment:
Any particular reason not to use `order`?

Reply:
Same as above.